src/components/MyAccount/RequestsTab/RequestsTab.test.tsx

```diff
@@ -0,0 +1,169 @@
+import React from "react"
```

Comment:
These tests are really good but if possible, render the `RequestTab` component instead of `ProfileTabs`. You can use `useState` in test files so you can mock the `handleHoldsState` function.